.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -8,7 +8,7 @@ body:
         Before you post this issue, please check the documentation:
 
         - [nf-core website: troubleshooting](https://nf-co.re/usage/troubleshooting)
-        - [nf-core/staramr pipeline documentation](https://nf-co.re/staramr/usage)
+        - [phac-nml/staramrnf pipeline documentation](https://https://github.com/phac-nml/staramrnf/blob/main/docs/usage.md)
```

Comment:
You have an extra `https://` in the URL here.

Reply:
9c52b62